inspirehep/modules/workflows/workflows/article.py

```diff
@@ -76,6 +76,7 @@
 from inspirehep.modules.workflows.tasks.upload import store_record, set_schema
 from inspirehep.modules.workflows.tasks.submission import (
     add_note_entry,
+    add_internal_uid,
```

Comment:
There's a problem with this (even if your approach is still better): integration tests for `formdata_to_model` will need to hardcode some user id in the fixture, but it's not guaranteed to be constant across multiple runs of the tests. This is a limitation of the design of the tests of `formdata_to_model`...